src/worker.cc

```
@@ -1130,6 +1130,14 @@ void from_json(rapidjson::Document& doc, Options::Action action, Api& api) {
   // if specified, get the show_locations boolean in there
   options.set_show_locations(rapidjson::get<bool>(doc, "/show_locations", options.show_locations()));
 
+  // if specified, get the isochrone_grid_size in there
+  auto isochrone_grid_size = rapidjson::get_optional<float>(doc, "/isochrone_grid_size");
```

Should I `throw valhalla_exception_t{#}` if the value is invalid?